From c47afa7e82a50544a4801d60a98255b30f2d764f Mon Sep 17 00:00:00 2001
From: Akshu Agrawal <akshu.agrawal@amd.com>
Date: Thu, 29 Jun 2017 11:44:52 +0530
Subject: [PATCH 2393/2500] sound/amd/: Changing ACPI ID

Changing ACPI ID to AMDI1002 to be inline with convention

Reviewed-by: Alex Deucher <alexander.deucher@amd.com>
Signed-off-by: Akshu Agrawal <akshu.agrawal@amd.com>
Signed-off-by: Alex Deucher <alexander.deucher@amd.com>
---
 sound/soc/amd/acp-rt5645.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/amd/acp-rt5645.c b/sound/soc/amd/acp-rt5645.c
index ebaafad..a17dbea 100644
--- a/sound/soc/amd/acp-rt5645.c
+++ b/sound/soc/amd/acp-rt5645.c
@@ -189,7 +189,7 @@ static int cz_remove(struct platform_device *pdev)
 }
 
 static const struct acpi_device_id cz_audio_acpi_match[] = {
-	{ "I2SC1002", 0 },
+	{ "AMDI1002", 0 },
 	{},
 };
 
-- 
2.7.4

